cli/index.js

```diff
@@ -81,4 +431,425 @@ y.command({
     }
 })
 
+// create login command
+y.scriptName("connectme")
+y.command({
+    command: 'login',
+    describe: 'login with an account',
+    builder: {
+        username: {
+            describe: 'Username',
+            demandOption: true,
+            type: 'string'
+        },
+        password: {
+            describe: 'Password of the account'
+        }
+        // TODO: prompt to confirm the password
+    },
+    handler(argv) {
+        login(argv)
+    }
+})
+
+// create guest account command
+y.scriptName("connectme")
+y.command({
+    command: 'guest',
+    describe: 'Creates a guest account ',
+    builder: {
+        username: {
+            describe: 'Username',
+            demandOption: true,
+            type: 'string'
+        },
+        userID: {
+            describe: 'An user ID',
+            demandOption: true,
+            type: 'string'
+        },

```

Comment:
you cannot specify the userId at creation, it doesn't make sense. This is similar to use signUp, so I would copy from there what you need

Reply:
done